teams.md

```diff
@@ -125,6 +125,17 @@ Recent projects:
 * Allow users to customize terminal fonts and keyboard shortcuts
 * Add a new cross-cluster search experience
 
+### Access Graph
+
+The Access Graph team specializes in analyzing resource access patterns, 
+providing valuable insights and enhanced visibility into infrastructure 
+by examining various access paths.
+
+Recent projects:
+
+* Import of Teleport RBAC and AWS IAM into a unified representation.
+* Implementing SQL-like query language for querying the access paths.
```

Comment:
I would mention something about UI here as well as this is where most innovation is happening it feels like, something like "Designing scalable graph UI that can display hundreds of resources while maintaining responsiveness and user-friendliness."